### DISCUSSION TITLE
Issue with Github pages

### DISCUSSION THREAD

**Discussion by BaibhabRanjan** [community/community#63822] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Has anyone else encountered issues with pages not loading properly on GitHub? I've been experiencing difficulties accessing certain pages, and I'm wondering if it's just me or if there's a larger issue at play. Any insights or suggestions would be greatly appreciated.

![image](https://github.com/community/community/assets/134831003/d4bbdcb8-7fec-472d-a5f3-d167e7c43cae)

![image](https://github.com/community/community/assets/134831003/49cc3923-d84e-4a6b-9a37-752468e54ff7)
These are the files for the page

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @BaibhabRanjan , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. 

We worked with the Pages team to create a [guide](https://github.com/orgs/community/discussions/61073) to help you troubleshoot this error. Check out the [Pages 404 error guide](https://github.com/orgs/community/discussions/61073) for assistance. And don't forget to spread the knowledge 💡 and the guide to your fellow community members.

**Comment by radar-blip** [comment]
Rename `Index.Html` to `index.html` and it should work.

Also, there's going to be a problem with this link here `<link rel="stylesheet" href="style.css">`
because the `style.css` doesn't match the `Style.css` in your directory. 
I would suggest making both lowercase.

**Reply by BaibhabRanjan** [reply]
Thanks for your response David, I tried both the way you mentioned but still not able to load the page. Also I have tried with another repo, same problem arises.
![image](https://github.com/community/community/assets/134831003/613412a4-ede3-4a2a-8dba-711266199131)

**Reply by radar-blip** [reply]
The .Html extenstion must be .html

**Reply by BaibhabRanjan** [reply, marked as answer]
After changing from Html to html it is now working
Thank you so much

**Reply by radar-blip** [reply]
You're welcome.
